packages/types/index.d.ts

```diff
@@ -500,6 +500,58 @@ export interface IResultsOptions {
   };
 }
 
+export interface ICSVExportOptions {
+  /**
+   * Field delimiter
+   * @type {string}
+   * @default ','
+   * @memberof ICSVExportOptions
+   */
+  delimiter?: string;
+  /**
+   * Include header line
+   * @type {boolean}
+   * @default true
+   * @memberof ICSVExportOptions
+   */
+  header?: boolean;
+  /**
+   * Quote characters
+   * @type {string}
+   * @default '"'
+   * @memberof ICSVExportOptions
+   */
+  quote?: string;
+  /**
+   * Quote all non-empty fields
+   * @type {boolean}
+   * @default true
+   * @memberof ICSVExportOptions
+   */
+  quoted?: boolean;
+  /**
+   * Quote empty fields
+   * @type {boolean}
+   * @default true
+   * @memberof ICSVExportOptions
+   */
+  quoted_empty?: boolean;
+  /**
+   * Quote all string fields
+   * @type {boolean}
+   * @default false
```

Comment:
Change this if you agree with my other comment about the default becoming `true` for back-compatibility.